android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -1339,6 +1361,8 @@ public void loadAppOpenAd(final String adUnitId)
         }
 
         MaxAppOpenAd appOpenAd = retrieveAppOpenAd( adUnitId );
+        if ( appOpenAd == null ) return;
```

Comment:
Is this LoC related to this change? If not, let's remove it and put into another PR.

Reply:
Removed.